### ISSUE TITLE
Add `ps` and `kill` commands for administrators

### ISSUE THREAD

**Issue by parsley42** [lnxjedi/gopherbot#137]
Output from `ps` should be e.g.:
```
wid  pwid pipename    taskname    type arguments
 100    0 localbuild  localbuild  ext  github.com/lnxjedi/gopherbot master
 102  100 localbuild  exec        ext  .gopherci/publish.sh gopherbot.tar.gz
 100    0 knock       knock       int
```
`kill` should:
* Mark the killed pipeline as cancelled under lock
* If type  == ext (external) send a kill -9 to the external task
* Leave children alone; don't get in to the business of hunting down and killing child jobs
* If the current task type == int, give an error message that Go tasks can't be killed

Implementation:
* `runtasks` should check the value of `Cancelled` before starting a new task
* Cancelled pipelines should return a new cancelled error value
* Commands should only work for robot administrators, and only in the robot's `DefaultJobChannel`

**Comment by parsley42** [comment]
Added!
